examples/spi-demo/src/main.rs

```diff
@@ -63,11 +63,11 @@ fn main() -> ! {
 
     // This will write 8 bytes, then shift out ORC
 
-    // Note :     spi.read( &mut cs.degrade(), reference_data, &mut readbuf )
-    //            will fail because reference data is in flash, the copy to
-    //            an array will move it to RAM.
+    // Note: `spi.transfer_split_uneven(&mut cs.degrade(), reference_data, &mut readbuf)`
+    //       will fail because reference data is in flash, the copy to an array
+    //       will move it to RAM.
```

Comment:
Is this limitation and comment still valid?

Reply:
Should be, yes